app/views/devise/passwords/edit.html.haml

```diff
@@ -1,19 +1,24 @@
-%h2 Change your password
-= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
-  = devise_error_messages!
-  = f.hidden_field :reset_password_token
-  .field
-    = f.label :password, 'New password'
-    %br/
-    - if @minimum_password_length
-      %em
-        (#{@minimum_password_length} characters minimum)
-      %br/
-    = f.password_field :password, autofocus: true, autocomplete: 'off'
-  .field
-    = f.label :password_confirmation, 'Confirm new password'
-    %br/
-    = f.password_field :password_confirmation, autocomplete: 'off'
-  .actions
-    = f.submit 'Change my password'
-= render 'devise/shared/links'
+.h1-top
+  %h1
+    Change your password
+.container
+  .row
+    .col-md-8.col-xs-12
+      = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }, class: 'form-group') do |f|
+        = devise_error_messages!
+        = f.hidden_field :reset_password_token, class: 'form-control'
+        .form-group
+          = f.label :password, 'New password'
+          %br/
+          - if @minimum_password_length
+            %em
+              (#{@minimum_password_length} characters minimum)
```

Comment:
Hmm... Are we using `@minimum_password_length` in this app? Can't remember any scenario for that.